wallet/README.recover.md

```diff
@@ -121,6 +121,15 @@ succeeds, it will print the transaction hash of the created transaction. You can
 a block explorer to watch, or just open the destination wallet and watch from there. If your coins
 are confirmed, you're done and you can skip the next paragraph to EPILOGUE.
 
+You can also get a list of the private keys, which would be useful if you want to sweep them into a wallet (for example a Bitcoin Cash wallet). If your wallet had a spending PIN set you need to decrypt it first, otherwise the private keys won't appear:
+
+    ./wallet-tool decrypt --wallet=/tmp/bitcoin-wallet-decrypted-backup --password=<spending-PIN>
+
```

Comment:
That's not ideal from a security perspective, since it leaves your decrypted wallet lying around.

It's better to skip it and append `--password=<spending-PIN>` to your second command line below, but that feature has only been added recently, so it's only available on the master branch.

Reply:
Only available in bitcoinj master I understand. Then we would need to update the `git clone` command above, and retest the whole recovery procedure in the new version, but we shouldn't do that until bitcoinj releases.

I suggest I add a note about this security issue (it could be done in an offline computer, for example) and then we can update in the future once the new bitcoinj version is released.

Reply:
Ok makes sense.

Reply:
@schildbach I updated the documentation with this security concern, please review it.